src/amo/components/Categories.scss

```diff
@@ -10,11 +10,19 @@
   list-style: none;
   margin: 0 auto;
   padding: 0;
+
+  [dir=rtl] & {
+    transform: scale(-1, 1);
```

Comment:
We actually don't need to scale the entire element; that's more effort on the renderer and could change other content. We should be changing the background element used and changing its position. eg:

```scss
[dir=rtl] & {
  background-image: url('../img/icons/arrow-left.svg');
  background-position-x: 0%;
}
```

You'll need to create an inverted arrow icon if we don't have one. Shame we aren't using an icon component here...